core/crypto/bls_provider/bls_provider_impl.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CRYPTO_BLS_PROVIDER_IMPL_HPP
+#define CRYPTO_BLS_PROVIDER_IMPL_HPP
+
+#include "crypto/bls_provider/bls_provider.hpp"
+
+namespace fc::crypto::bls {
+  class BlsProviderImpl : public BlsProvider {
```

Comment:
Please add comments.

Reply:
TBH I don't think we need comments like:
`// Class implementing BlsProvider`
It is enough to have comments only for BlsProvider unless BlsProviderImpl contains some new methods that are not inherited from the parent class